manager-bundle/src/Routing/RouteLoader.php

```diff
@@ -89,17 +89,25 @@ function (RouteCollection $collection, RoutingPluginInterface $plugin): RouteCol
         return $collection;
     }
 
-    private function getConfigFile(string $file): ?string
+    private function getConfigFile(): ?string
     {
-        if (file_exists($this->rootDir.'/config/'.$file)) {
-            return $this->rootDir.'/config/'.$file;
+        foreach (['routes.yaml', 'routes.yml', 'routing.yaml', 'routing.yml'] as $file) {
+            if (file_exists($this->rootDir.'/config/'.$file)) {
+                if (0 === strncmp($file, 'routing.', 8)) {
+                    @trigger_error(sprintf('Using a "%s" file has been deprecated and will no longer work in Contao 5.0. Rename it to "routes.yaml" instead.', $file), E_USER_DEPRECATED);
+                }
+
+                return $this->rootDir.'/config/'.$file;
+            }
         }
 
         // Fallback to the legacy config file (see #566)
-        if (file_exists($this->rootDir.'/app/config/'.$file)) {
-            @trigger_error(sprintf('Storing the "%s" file in the "app/config" folder has been deprecated and will no longer work in Contao 5.0. Move it to the "config" folder instead.', $file), E_USER_DEPRECATED);
+        foreach (['routing.yaml', 'routing.yml'] as $file) {
```

Comment:
`routes.y(a)ml` should be supported in the old place as well

Reply:
Really? I have just asked in the Mumble call and @ausi and @Toflar share my opinion that the new Symfony 4 file does not have to be supported in the old Symfony 3 config location.

Reply:
I only remember we said „keep it consistent everywhere“? 🤷‍♂️

Reply:
Both true :-)

Reply:
Yes, keep it consistent as in "allow .yml and .yaml" everywhere. But not "allow Symfony 4 files in the deprecated Symfony 3 file location".

Reply:
/cc @contao/developers

Reply:
I agree with @leofeyer

Reply:
Auf mehrfachen Wunsch eines einzelnen: a1a5332c549a6f108ae3f2004e25c3cee9c2b16e 🤷‍♂